test/services/permissions.test.js

```diff
@@ -826,14 +826,14 @@ describe('services > permissions', () => {
                   collections: {
                     Users: {
                       collection: {
-                        browseEnabled: true,
+                        addEnabled: true,
                       },
                     },
                   },
                 },
               });
 
-            await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 }))
+            await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'addEnabled', { userId: 1 }))
```

Comment:
I changed this value to prevent having to mock the route with the `renderingSpecificOnly` parameter.